app/controllers/reminder_mailings_controller.rb

```diff
@@ -0,0 +1,87 @@
+class ReminderMailingsController < ApplicationController
+  before_action :set_reminder_mailing, only: [:show, :edit, :update, :destroy, :initiate_mailing]
+
+  def index
+    authorize ReminderMailing
+    @reminder_mailings = ReminderMailing.order('created_at desc').all
+  end
+
+  def show; end
+
+  def new_probation_period
+    @assignments = Assignment.need_probation_period_reminder_mailing.distinct
+    @group_assignments = GroupAssignment.need_probation_period_reminder_mailing.distinct
+    @reminder_mailing = ReminderMailing.new(kind: 'probation_period',
+      reminder_mailing_volunteers: @assignments + @group_assignments)
+
+    # TODO: load email default template from not yet existint EmailTemplate Model
+    @reminder_mailing.assign_attributes({
+      subject: "Errinnerung fuer %{Einsatz}",
+      body: "Hallo %{Anrede} %{Name}\r\n\r\n\r\n\r\n%{Einsatz} gestarted am %{EinsatzStart}"
+    })
+    authorize @reminder_mailing
+  end
+
+  def new_half_year
+    @reminder_mailables = Assignment.started_six_months_ago + GroupAssignment.started_six_months_ago
+    @reminder_mailing = ReminderMailing.new(kind: 'half_year', reminder_mailing_volunteers: @reminder_mailables)
+
+    # TODO: load email default template from not yet existint EmailTemplate Model
+    @reminder_mailing.assign_attributes({
+      subject: "Errinnerung fuer %{Einsatz}",
+      body: "Hallo %{Anrede} %{Name}\r\n\r\n\r\n\r\n%{Einsatz} gestarted am %{EinsatzStart}"
+    })
+    authorize @reminder_mailing
+  end
+
+  def create
+    @reminder_mailing = ReminderMailing.new(reminder_mailing_params)
+    @reminder_mailing.creator = current_user
+    authorize @reminder_mailing
+    if @reminder_mailing.save
+      redirect_to @reminder_mailing, make_notice
+    else
+      render "new_#{@reminder_mailing.kind}".to_sym
+    end
+  end
+
+  def edit; end
+
+  def initiate_mailing
+    if @reminder_mailing.sending_triggered
+      return redirect_to reminder_mailings_path, notice: 'Dieses Mailing wurde bereits versandt'
+    end
+    @reminder_mailing.reminder_mailing_volunteers.each do |mailing_volunteer|
+      VolunteerMailer.probation_period_reminder(mailing_volunteer).deliver_later
```

Comment:
what does `deliver_later` do? and where does it come from?

Reply:
clarified

Reply:
It's available on all mailers and enqueues a job to send the mail rather than sending it directly, see http://api.rubyonrails.org/classes/ActionMailer/MessageDelivery.html#method-i-deliver_later

Note that the default job queue in Rails will lose jobs after a server restart, for mails like this it should be ok but for more important jobs something like Sidekiq should be used.